src/view/widgets/accordion_widget.py

```diff
@@ -136,14 +139,19 @@ def get_multi_select_widget_from_row_with_tooltip(self, widget):
         :return: The multi-select widget if it matches the conditions, otherwise None.
         """
         if (
-            isinstance(widget, pn.Row)
-            and len(widget) == 2
-            and isinstance(widget[0], panel.widgets.select.MultiSelect)
+                isinstance(widget, pn.Row)
+                and len(widget) == 2
+                and isinstance(widget[0], panel.widgets.select.MultiSelect)
         ):
             return widget[0]
         else:
             return None
 
+    def on_accordion_change2(self, event):
+        # You can access the new value with event.new and the old value with event.old
+        print("Accordion state changed!")
```

Comment:
logging would be more appropriate if this is relevant information, if these are for debugging purposes, please remove them

Reply:
Done.